tim_mendoza/models/movie.js

```diff
@@ -0,0 +1,13 @@
+var mongoose = require('mongoose');
+var currentYear = new Date().getFullYear();
```

Comment:
Imagine the year changed after you deployed this project. It might not update this variable until you reset the server or something. How can you make sure this uses the correct year dynamically?